tests/endtoend/test_eventgrid_functions.py

```diff
@@ -69,17 +86,82 @@ def test_eventgrid_trigger(self):
                 response = r.json()
 
                 self.assertEqual(
-                    response,
-                    {
-                        'id': data[0]['id'],
-                        'data': data[0]['data'],
-                        'topic': data[0]['topic'],
-                        'subject': data[0]['subject'],
-                        'event_type': data[0]['eventType'],
-                    }
+                    response, {'id': data[0]['id'], 'data': data[0]['data'],
+                               'topic': data[0]['topic'],
+                               'subject': data[0]['subject'],
+                               'event_type': data[0]['eventType']}
                 )
             except AssertionError:
                 if try_no == max_retries - 1:
                     raise
             else:
                 break
+
+    @testutils.retryable_test(1, 5)
+    def test_eventgrid_output_binding(self):
+        """test event_grid output binding
+
+        This test needs three functions to work.
+        1. `eventgrid_output_binding`
+        2. `eventgrid_output_binding_message_to_blobstore`
+        3. `eventgrid_output_binding_success`
+
+        This test calls the eventgrid_output_binding function, sends in a unique
+        uuid as `data` in the body to the httpTrigger which sends in that value
+        in the eventGrid output data. The eventGrid topic is configured to
+        send the event to a storage queue.
+
+        The second function (`eventgrid_output_binding_message_to_blobstore`)
+        reads from that storage queue and puts into a blob store.
+
+        The third function (`eventgrid_output_binding_success`) reads the
+        text from the blob store and compares with the expected result. The
+        unique uuid should confirm if the message went through correctly to
+        EventGrid and came back as a blob.
+        """
+
+        test_uuid = uuid.uuid4().__str__()
+
+        data = "{" + "'test_uuid': '{0}'".format(test_uuid) + "}"
```

Comment:
Using f"string" would be cleanner in this case (Python >= 3.6)
```python
data = f"{{ 'test_uuid': '{test_uuid}' }}"
```

Reply:
Thanks. Yes, I tried that earlier inside the `format()` in the next line, but that was throwing some other issue. Fixed it.